pkg/controller/featuregates_hook.go

```diff
@@ -64,7 +65,10 @@ func UpdateDeploymentFeatureGatesHook(
 			if !strings.EqualFold(deployment.Spec.Template.Spec.Containers[i].Name, "manager") {
 				continue
 			}
-			if err := setContainerFeatureGateArg(&deployment.Spec.Template.Spec.Containers[i], featureGatesToSet); err != nil {
+
+			// If featureSet is not found in AllFixedFeatureSets, then it Custom, so we need to ignore mismatches
```

Comment:
```suggestion
			// If featureSet is not found in AllFixedFeatureSets, then it is Custom, so we need to ignore mismatches
```
?
Is there any such thing as a "custom feature set", or are you just covering all the bases?